packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist.res

```diff
@@ -168,6 +168,22 @@ let pdf = (f: float, t: t) => {
 let inv = T.Integral.yToX
 let cdf = T.Integral.xToY
 
+let diff = (arr: array<float>): array<float> =>
+  Belt.Array.zipBy(arr, Belt.Array.sliceToEnd(arr, 1), (left, right) => right -. left)
+
+let rec rangeByFloat = (start : float, end: float, step: float) => 
```

Comment:
Should go in `E.Float` module.

Reply:
Yes it should!

Reply:
Lol, thanks for telling me this, because it turns out rangeByFloat is already in the codebase in E

Reply:
actually check to see if it already is. Ozzie or I might have put it in there during CR of the prior sparklinesy PR.